src/cryptonote_basic/cryptonote_boost_serialization.h

```diff
@@ -71,7 +71,11 @@ namespace boost
   {
     a & reinterpret_cast<char (&)[sizeof(crypto::key_image)]>(x);
   }
-
+  template <class Archive>
+  inline void serialize(Archive &a, crypto::view_tag &x, const boost::serialization::version_type ver)
+  {
+    a & reinterpret_cast<char (&)[sizeof(crypto::view_tag)]>(x);
```

Comment:
With this complicated construct instead of `a & x.data;` do you want to be prepared for view tags that are possibly more than 1 character in the future?

Reply:
Was following the pattern for the other crypto classes right above, I figure deviating from the pattern is more complicated than not

I'd guess the intention with the pattern is to avoid referencing `data` to maintain a higher level abstraction when working with the crypto classes across the code base (there seems to be a conscious effort *not* to reference `data` with the others in other places in the code), but I don't know for sure